ocs_ci/ocs/platform_nodes.py

```
@@ -4249,3 +4262,504 @@ def disable_nodes_network_temporarily(self, nodes, duration=20):
+                f"Known nodes: {list(self._agent_node_map.keys())}"
+            )
+
+        manufacturer = details["manufacturer"].lower()
+        rack_ip = details["rack_ip"]
+        username = details["username"]
+        password = details["password"]
+
+        if not all([rack_ip, username, password]):
+            raise RuntimeError(
+                f"Missing rack IP or credentials for agent node {node_name}"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)